### ISSUE TITLE
Where did this come from?

### ISSUE THREAD

**Issue by Anonymous941** [historicalsource/sinistar#2]
I'm curious, is this a leak or officially released, and where did you find it?

**Comment by c3s0fr3sh** [comment]
No idea but curious on if someone actually can build it! Working on a music project and would love to integrate something! But I don’t code lol.

**Comment by Anonymous941** [comment]
That's what I've been trying to do in #1, but I'm not very good with ASM...  Although the comments are really interesting, this is the best I've seen in any project actually.  Also, I'm not sure that even if it was built you could embed custom music without making a custom utility and composing music with it, although I don't know if there's some universal format

**Comment by c3s0fr3sh** [comment]
Damn. Yeah, i know nothing about coding definitely not my skill in life but am an avid game enthusiast probably could get into wwise but like you said it may be an impossible feat.

**Comment by Anonymous941** [comment]
> Damn. Yeah, i know nothing about coding definitely not my skill in life but am an avid game enthusiast probably could get into wwise but like you said it may be an impossible feat.

Yeah, getting custom assets into ancient programs (even with source code) is annoyingly difficult.  I recommend reading the comments if you're interested in Sinistar though, they're pretty interesting even if you're not a programmer (especially WITT/WARRIOR.SRC)
